shoop/front/checkout/addresses.py

```diff
@@ -31,6 +31,11 @@ def __init__(self, **kwargs):
             # Set default country
             self.fields["country"].initial = settings.SHOOP_ADDRESS_HOME_COUNTRY
 
+        required_address_fields = set(settings.SHOOP_FRONT_ADDITIONAL_REQUIRED_ADDRESS_FIELDS)
```

Comment:
Wouldn't it be better to just allow overriding any sort of property on the fields?

```python
SHOOP_FRONT_ADDRESS_FIELD_PROPERTIES = {
  "phone": {"required": False, "help_text": "But who was phone?"}
}
```

That would make it possible to make fields that are required as built-in non-required, etc.